packages/pigment-css-react/README.md

```diff
@@ -306,11 +307,12 @@ const Button = styled('button')({
 
 #### Styling based on runtime values
 
-> 💡 This approach is recommended when the value of a prop is **unknown** ahead of time or possibly unlimited values, for example styling based on the user's input.
+> 💡 This approach is recommended when the values of the props are not known ahead of time or are potentially unlimited—for example, styling based on the user's input.
 
-There are two ways to acheive this (both involve using a CSS variable):
+There are two ways to apply styles based on runtime values.
+Both involve using a CSS variable:
 
-1. Declare a CSS variable directly in the styles and set its value using inline styles:
+**Option 1** – Declare a CSS variable directly in the styles and set its value using inline styles:
 
 ```jsx
 const Heading = styled('h1')({
```

Comment:
We have `const Heading` and `function Heading` in this example so we need to rename one.